index.html

```diff
@@ -79,18 +79,14 @@ <h3>
         </div>
      </main>
 
- 
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
 
     <!--
-
     Hello Camper!
-
     For now, the test suite only works in Chrome!
     Please read the README below in the JS Editor before beginning.
-    Feel free to delete this message once you have read it.
-    Good luck and Happy Coding!
-
+    Feel free to delete this message once you have read it..
+    Good luck and Happy Coding! 
     - The freeCodeCamp Team
 
     -->

```

Comment:
- [ ]  Kindly remove all inline comments from your code.

Reply:
It's highly recommended to use suggestions sentences when you perform a code review

Good work implementing that in your CR 😊